content/en/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md

```diff
@@ -24,9 +26,10 @@ vacated by the evicted critical add-on pod or the amount of resources available
 
 ### Marking pod as critical
 
-To be considered critical, the pod has to run in the `kube-system` namespace (configurable via flag) and
+Prior to v1.11, critical pod has to run in the `kube-system` namespace, this restriction was removed after v1.11 and pod in any namespace can be configed as a critical pod by the following either way:
```

Comment:
Critical pods shouldn't be restricted to kube-system #60653
https://github.com/kubernetes/kubernetes/pull/60653

Reply:
“and pod in any namespace can be configed as a critical pod by the following either way:”

This statement is verbose and contains a few grammar errors.